docs/content/doc/administration/config-cheat-sheet.en-us.md

```diff
@@ -465,6 +465,7 @@ relation to port exhaustion.
 - `ISSUE_INDEXER_PATH`: **indexers/issues.bleve**: Index file used for issue search; available when ISSUE_INDEXER_TYPE is bleve and elasticsearch. Relative paths will be made absolute against _`AppWorkPath`_.
 
 - `REPO_INDEXER_ENABLED`: **false**: Enables code search (uses a lot of disk space, about 6 times more than the repository size).
+- `REPO_INDEXER_UNITS`: **sources,forks,mirrors,templates**: Repo indexer units. The items to index could be `repos`, `forks`, `mirrors`, `templates` or any combination of them separated by a comma. If empty then it defaults to `sources` only, as if you'd like to disable fully please see `REPO_INDEXER_ENABLED`.
```

Comment:
The ini says: `;REPO_INDEXER_REPO_TYPES = sources,forks,mirrors,templates`

Here says: `repos`, `forks`, `mirrors`, `templates`